### PR TITLE
Feature/aboutus

### PR DESCRIPTION
**Pull request by MrSlesh**
aboutus new

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -22,6 +23,7 @@ function App(): JSX.Element {
 					<Route path="payment" element={<Payment />} />
 					<Route path="return" element={<Return />} />
 					<Route path="support" element={<Support />} />
+					<Route path="*" element={<AboutUs />} />
```
Comment (Igor-Klein-1988) [inline src/App.tsx:26]: path="aboutus"

File: src/pages/AboutUs.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+.about-us-header {
+  font-size: 24px;
+  font-weight: bold;
+  color: #333; /* Цвет текста, можно заменить на нужный вам */
+  margin-bottom: 10px; /* Отступ снизу */
```
Comment (Igor-Klein-1988) [inline src/pages/AboutUs.css:12]: Delete all comments:)

File: src/pages/AboutUs.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { MapContainer as LeafletMapContainer, TileLayer, Marker, Popup } from 'react-leaflet';
+import { LatLngExpression } from 'leaflet';
+import 'leaflet/dist/leaflet.css';
```
Comment (Igor-Klein-1988) [inline src/pages/AboutUs.tsx:4]: Add empty line.
Do you use this?

File: src/pages/AboutUs.tsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+	const companyLocation: LatLngExpression = [52.462894, 13.508654];
+
+	return (
+		<div className="about-us-container">
+			<div className="about-us-header">
+				<h1>О Нас</h1>
+			</div>
+			<div className="about-us-content">
+				<p>
+					Добро пожаловать в Allcom, ваш первоклассный выбор для захватывающих аукционных продаж. В
```
Comment (Igor-Klein-1988) [inline src/pages/AboutUs.tsx:17]: Create conts in i18n

File: src/pages/AboutUs.tsx [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+					Allcom мы специализируемся на объединении покупателей и продавцов в динамичном и
+					прозрачном рынке. Наша команда страстно относится к объединению людей с уникальными и
+					ценными предметами через онлайн аукцион.
+				</p>
+				<p>
+					Независимо от того, являетесь ли вы опытным коллекционером, ищущим редкие артефакты, или
```
Comment (Igor-Klein-1988) [inline src/pages/AboutUs.tsx:23]: Same

File: src/pages/AboutUs.tsx [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+					продавцом, заинтересованным в привлечении глобальной аудитории, Allcom предоставляет
+					платформу, отвечающую вашим потребностям. Наши аукционы включают в себя разнообразные
+					категории, включая искусство, антиквариат, ювелирные изделия, винтажные товары и многое
+					другое.
+				</p>
+				<p>
+					Так же наша компания в скором времени запустит возможность доставки, об этом мы сразу
```
Comment (Igor-Klein-1988) [inline src/pages/AboutUs.tsx:30]: Same

File: src/pages/AboutUs.tsx [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+					сообщим на сайте.
+				</p>
+				<p>
+					Локация нашего склада указана на карте для удобства Вашего поиска. Находимся Мы в городе
```
Comment (Igor-Klein-1988) [inline src/pages/AboutUs.tsx:34]: Same

File: src/pages/AboutUs.tsx [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+					Berlin по адресу Wilhelminenhofstraße 93, 12459.
+				</p>
+				<div className="map-container">
+					<div className="map-wrapper">
+						<div className="leaflet-map-container">
+							<LeafletMapContainer
+								center={companyLocation}
+								zoom={13}
+								style={{ height: '300px', width: '100%' }}
```
Comment (Igor-Klein-1988) [inline src/pages/AboutUs.tsx:43]: Use css or scss

File: src/pages/AboutUs.tsx [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+import 'leaflet/dist/leaflet.css';
+import './AboutUs.css';
+
+const AboutUs: React.FC = (): JSX.Element => {
+	const companyLocation: LatLngExpression = [52.462894, 13.508654];
+
+	return (
+		<div className="about-us-container">
+			<div className="about-us-header">
+				<h1>О Нас</h1>
```
Comment (Igor-Klein-1988) [inline src/pages/AboutUs.tsx:13]: Use i18n

File: src/pages/AboutUs.tsx [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+				<p>
+					Локация нашего склада указана на карте для удобства Вашего поиска. Находимся Мы в городе
+					Berlin по адресу Wilhelminenhofstraße 93, 12459.
+				</p>
+				<div className="map-container">
+					<div className="map-wrapper">
+						<div className="leaflet-map-container">
+							<LeafletMapContainer
+								center={companyLocation}
+								zoom={13}
```
Comment (Igor-Klein-1988) [inline src/pages/AboutUs.tsx:42]: Magic number

File: src/pages/AboutUs.tsx [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+								style={{ height: '300px', width: '100%' }}
+							>
+								<TileLayer
+									url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
+									attribution='&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
+								/>
+								<Marker position={companyLocation}>
+									<Popup>Allcom</Popup>
```
Comment (Igor-Klein-1988) [inline src/pages/AboutUs.tsx:50]: Use i18n

File: src/App.tsx (hunk 1 above)
Comment (MrSlesh) [reply src/App.tsx:26]: corrected

File: src/pages/AboutUs.css (hunk 2 above)
Comment (MrSlesh) [reply src/pages/AboutUs.css:12]: corrected

File: src/pages/AboutUs.tsx (hunk 3 above)
Comment (MrSlesh) [reply src/pages/AboutUs.tsx:4]: corrected

leaflet necessary for the card to work

File: src/pages/AboutUs.tsx (hunk 10 above)
Comment (MrSlesh) [reply src/pages/AboutUs.tsx:42]: 12 corrected

**Pull request closed without merging**